typeclasses/mixins.py

```diff
@@ -66,6 +68,7 @@ def __temp_desc_set(self, val):
             self.db.raw_desc = self.db.desc
         if not self.db.general_desc:
             self.db.desc = self.db.desc
+        self.ndb.cached_string_after_template_replacement = None
```

Comment:
Whoops, I think I suggested the wrong variable name, doesn't match the attribute below. Should be:

```suggestion
        self.ndb.cached_template_desc = None
```

Reply:
done